### PR TITLE
Foreman 3.4

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: theforeman.org/yaml/jobs/test_3_4_stable.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          branch: '3.4-stable'
+    triggers:
+      - scm_fifteen_minutes
+      - github
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+            - 2.5
```
Comment (ekohl) [inline theforeman.org/yaml/jobs/test_3_4_stable.yaml:18]: We've dropped 2.5 support for 3.4.
```suggestion
```

File: theforeman.org/yaml/jobs/test_proxy_3_4_stable.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          branch: '3.4-stable'
+    triggers:
+      - scm_fifteen_minutes
+      - github
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+          - 2.5
```
Comment (ekohl) [inline theforeman.org/yaml/jobs/test_proxy_3_4_stable.yaml:18]: And here
```suggestion
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: theforeman.org/yaml/jobs/test_3_4_stable.yaml (hunk 1 above)
Comment (pcreech) [reply theforeman.org/yaml/jobs/test_3_4_stable.yaml:18]: Oh, nevermind.  I read this completely wrong and thought you were talking about something else.

Comment (pcreech) [reply theforeman.org/yaml/jobs/test_3_4_stable.yaml]: updated.

File: theforeman.org/yaml/jobs/test_proxy_3_4_stable.yaml (hunk 2 above)
Comment (pcreech) [reply theforeman.org/yaml/jobs/test_proxy_3_4_stable.yaml]: updated.

**Review by ekohl — APPROVED** [review]

**Pull request merged**